docs/architecture/adr-002-docs-structure.md

```diff
@@ -0,0 +1,270 @@
+
+|                 | Tutorials | How-to guides | Reference   | Explanation   |
+|-----------------|-----------|---------------|-------------|-------------- |
+| **Oriented to** | Learning  | A goal        | Information | Understanding | 
+| **Must**        | Allow a newcomer to get started | Show how to solve a specific problem | Describe the machinery | Explain |
+| **Takes the form of**    | A lesson | A series of steps | A dry description | A discursive explanation |
+| **Analogy**     | Teaching a child to cook | Recipe in a cookery book | An encyclopedia article | A paper on culinary social history |
+
+The specific implementation for Cosmos Cash SHOULD BE as per the following tree structure.
+
+```
+/
+├── README
+├── CONTRIBUTING
+├── TECHNICAL-SETUP
+├── CODEOWNERS
+├── x/
+|   ├── module_a/
+|       ├── README
+|       ├── docs/
+|           ├── state
+|           ├── state_transitions
+|           ├── messages
+├── docs/
+    ├── README
+    ├── CODEOWNERS
+    ├── Explanation/
+    |   ├── README
+    |   ├── ADR/
+    |   |   ├── README
+    |   |   ├── PROCESS
+    |   |   ├── adr-template
+    |   |   ├── adr-{number}-{desc}
+    |   ├── articles/
+    |   |   ├── regulation-litepaper/
+    |   |       ├── ARTICLE
+    |   ├── research/
+    |       ├── README
+    |       ├── research_topic/
+    ├── How-To/
+    |   ├── HowToDoSomething/
+    |   ├── HowToDoSomethingElse/
+    ├── Reference/
+    |   ├── README
+    |   ├── GLOSSARY
+    |   ├── MODULES
+    |   ├── use-cases/
+    |   |   ├── use-case-A
+    |   |   ├── use-case-B
+    |   ├── architecture/
+    ├── Tutorials/
+        ├── Tutorial_1/
+        ├── Tutorial_2/
+```
+
+#### Root level documents
+
+Root level of the repo SHALL HAVE the following files:
+
+- **README.md** - use this for introduction and orientating the user. All README's SHOULD FOLLOW guidelines - see [GitHub guide](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-readmes)
+- **TECHNICAL-SETUP.md** - how to get started with the repo (could be a link to a tutorial or How-To)
+    - Links to specific tooling set-up - development tools, linters etc
+    - Dependencies such as [pre-commit](https://pre-commit.com/), 
+    - Building the code
+    - Running tests
+- **CONTRIBUTING.md** - This details how to new users can contribute to the project. In specific,
+    - Committing changes
+    - Commit message formats (see below)
+    - Raising PR's
+    - Code of Conduct
+- **CODEOWNERS** - although not part of documentation it will define the code maintainers who responsible for quality assurance on comments, PR's and issues.
+
+#### Modules
+
+In line with Cosmos SDK convention (TODO: needs reference) each module contains its own relevant documentation:
+
+- **Module specifications** - these are document that outline state transitions x/module-name/docs/
+- **Module-level README.md** e.g. x/module-name/README.md
```

Comment:
So far, I haven't noticed each module on the SDK repo using a module-specific README but I like the idea. I know we talked about this option and it makes sense.